### PR TITLE
Update from auto-update/renku-graph-2.22.2

### PR DESCRIPTION
**Pull request by RenkuBot**
This is an automated pull request.

/deploy #persist

### CODE REVIEW

File: CHANGELOG.rst [hunk 1]
```diff
@@ -2,18 +2,22 @@
 
 0.19.2
 ------
-This minor bugfix release comes with internal changes to the Knowledge Graph and a small fix for the Lineage API.
+This minor bugfix release comes with internal changes to the Knowledge Graph and a other bug fixes.
```
Comment (rokroskar) [inline CHANGELOG.rst:5]:
```suggestion
This minor bugfix release comes with internal changes to the Knowledge Graph and a bug fix on the core service.
```

File: CHANGELOG.rst [hunk 2]
```diff
@@ -2,18 +2,22 @@
...
 ------
-This minor bugfix release comes with internal changes to the Knowledge Graph and a small fix for the Lineage API.
+This minor bugfix release comes with internal changes to the Knowledge Graph and a bug fix on the core service.
 
 Bug fixes
 ~~~~~~~~~~
 
 * **Knowledge Graph**: Lineage API to read data from the new dataset in the Triples Store
+* **Knowledge Graph**: migration fixing creation dates on Plans in the Triples Store
+* **Renku core service**: a bug causing Plan creation date to be after Activity start date
```
Comment (lorenzo-cavazzi) [inline CHANGELOG.rst:12]: Should we always start these sentences with a verb? I'm not sure we have a fixed rule for that, but that's what we tired to do lately. Feel free to reword any of the suggested sentences

```suggestion
* **Knowledge Graph**: read lineage data from the new dataset in the Triples Store
* **Renku core service**: prevent Plans creation dates from preceding their corresponding Activities
* **Knowledge Graph**: fix wrong Plans creation dates through migration
```

File: CHANGELOG.rst [hunk 3]
```diff
@@ -2,18 +2,22 @@
 
 0.19.2
 ------
-This minor bugfix release comes with internal changes to the Knowledge Graph and a small fix for the Lineage API.
+This minor bugfix release comes with internal changes to the Knowledge Graph and a bug fix on the core service.
```
Comment (lorenzo-cavazzi) [inline CHANGELOG.rst:5]:
```suggestion
This minor bugfix release comes with internal changes to the Knowledge Graph and fixes a glitch on the core service.
```

**Review by lorenzo-cavazzi — CHANGES_REQUESTED** [review]

**Review by lorenzo-cavazzi — APPROVED** [review]
Thank you for the quick fix! :rocket:

**Review by Panaetius — APPROVED** [review]

**Pull request merged**